Assets/Plugins/Editor/Uplift/DependencyResolution/DependencyNode.cs

```diff
@@ -136,5 +148,199 @@ public int Lowlink
 				lowlink = value;
 			}
 		}
+
+		private void UpdateNodeRequirements(DependencyDefinition newRequirements, string restrictor)
+		{
+			Debug.Log("Updating " + this.name + " requirement " + requirement);
```

Comment:
what happens if this is called while the requirements are already conflicting on that node? Is it possible?